backend/src/test/java/com/woowacourse/f12/application/product/ProductServiceTest.java

```diff
@@ -161,4 +114,25 @@ class ProductServiceTest {
                                 ETC_CONSTANT, 0.5))
         );
     }
+
+    @Test
+    void 제품명과_카테고리로_제품_목록을_조회한다() {
+        // given
+        Product product = KEYBOARD_1.생성(1L);
+        Pageable pageable = PageRequest.of(0, 2);
+        given(productRepository.findBySearchConditions("키보드1", KEYBOARD, pageable))
+                .willReturn(new SliceImpl<>(List.of(product), pageable, false));
+        ProductSearchRequest productSearchRequest = new ProductSearchRequest("키보드1", KEYBOARD_CONSTANT);
+
+        // when
+        ProductPageResponse productPageResponse = productService.findBySearchConditions(productSearchRequest, pageable);
+
+        // then
+        assertAll(
+                () -> verify(productRepository).findBySearchConditions("키보드1", KEYBOARD, pageable),
```

Comment:
```suggestion
                () -> verify(productRepository).findBySearchConditions(product.getName(), product.getCategory(), pageable),
```
product 를 활용해보는건 어떨까요?

Reply:
위의 코멘트와 동일한 내용입니다! :)